### PR TITLE
Akka Android (Do Not Merge)

### PR DESCRIPTION
**Pull request by annymsMthd**
This PR is to discuss the proper way to add android support for akka.net

### CODE REVIEW

File: src/core/Akka/Configuration/ConfigurationFactory.cs [hunk 1]
```diff
@@ -55,6 +55,7 @@ public static Config ParseString(string hocon)
             return ParseString(hocon, null);
         }
 
+#if !ANDROID
```
Comment (Aaronontheweb) [inline src/core/Akka/Configuration/ConfigurationFactory.cs]: Is it only the configuration stuff that needed to be  `IFDEF`-ed out in order for this to work?

Comment (annymsMthd) [inline src/core/Akka/Configuration/ConfigurationFactory.cs]: Pretty much. I'll be picking this up again within the next few days, it's
in our sprint

**Comment by annymsMthd** [comment]
@Aaronontheweb the demo app works on this now. I'm creating a nuspec for this project now.

**Comment by annymsMthd** [comment]
@Aaronontheweb We need to figure out a nuspec file that will bring this in with the main akka core package but targeting monoandroid

**Comment by Horusiath** [comment]
Maybe we should create a separate branch for this one?

**Pull request closed without merging**